xml/libvirt_configuration_virsh.xml

```diff
@@ -0,0 +1,1231 @@
+    <title>Configuring VM Guest to Use VF from the Pool</title>
+    <para>
+     The following example of guest device interface definition uses a VF of
+     the <xref linkend="vt.io.sriov"/> device from the pool created in
+     <xref linkend="libvirt.config.io.pool.host"/>. <systemitem class="library">libvirt</systemitem> automatically
+     derives the list of all VFs associated with that PF the first time the
+     guest is started.
+    </para>
+<screen>&lt;interface type='network'&gt;
+  &lt;source network='passthrough'&gt;
+&lt;/interface&gt;</screen>
+    <para>
+     To verify the list of associated VFs, run <command>virsh net-dumpxml
+     passthrough</command> on the host after the first guest that uses the
+     network with the pool of VFs starts.
+    </para>
+<screen>&lt;network connections='1'&gt;
+  &lt;name&gt;passthrough&lt;/name&gt;
+  &lt;uuid&gt;a6a26429-d483-d4ed-3465-4436ac786437&lt;/uuid&gt;
+  &lt;forward mode='hostdev' managed='yes'&gt;
+    &lt;pf dev='eth0'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x10' function='0x1'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x10' function='0x3'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x10' function='0x5'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x10' function='0x7'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x11' function='0x1'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x11' function='0x3'/&gt;
+    &lt;address type='pci' domain='0x0000' bus='0x02' slot='0x11' function='0x5'/&gt;
+  &lt;/forward&gt;
+  &lt;/network&gt;</screen>
+   </sect3>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="libvirt.config.storage.virsh">
+  <title>Configuring Storage Devices</title>
+  <para>
+  Information about storage devices is defined under the <tag>disk</tag> element.
+  The usual <tag>disk</tag> element supports several attributes, but the
+   following two attributes are the most important:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>The <tag class="attribute">type</tag> attribute describes the source
+     of the virtual disk device. Valid values are <tag class="attvalue">file</tag>,
+     <tag class="attvalue">block</tag>, <tag class="attvalue">dir</tag>,
+     <tag class="attvalue">network</tag>, or <tag class="attvalue">volume</tag>.</para>
+   </listitem>
+   <listitem>
+    <para>The <tag class="attribute">device</tag> attribute indicates, how
+     the disk is exposed to the guest OS. As an example, possible values
+     can include <tag
+      class="attvalue">floppy</tag>, <tag class="attvalue">disk</tag>,
+     <tag class="attvalue">cdrom</tag>, and others.
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   The following child elements are the most important:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para><tag>driver</tag> contains the driver and the bus, which is used by
```

Comment:
split in two sentences: '<tag>driver</tag> contains the driver and the bus. Those are used by the VM guest to work with the new disk device.</para>'